CardScan/Classes/SSDOcrOutputExtensions.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  SSDOcrOutputExtensions.swift
+//  CardScan
+//
+//  Created by xaen on 3/22/20.
+//
+
+import Foundation
+import Accelerate
```

Comment:
what is this library? Is it a standard iOS library? We probably need to add something to the podspec file to make sure that the app always includes it

Reply:
This is a standard library that uses CPU's vector instructions just like Metal uses the GPU.
This is already in the SDK and we have been using this since August'19.
https://github.com/getbouncer/cardscan-ios/blob/master/CardScan/Classes/SSDOutputExtensions.swift